test/lib/storage-test.js

```diff
@@ -0,0 +1,73 @@
+/* global describe, beforeEach, it */
```

Comment:
Instead of adding `/* global ... */` in all the ./test/*.js files, we may just want to create a ./test/.eslintrc.json file with the following sauce:

```json
{
  "env": {
    "mocha": true
  },
  "rules": {
    "promise/avoid-new": "off",
    "promise/no-callback-in-promise": "off"
  }
}
```

Plus, then we can remove all the rogue `// eslint-disable-line ...` comments below.